src/libraries/System.IO.Hashing/pkg/System.IO.Hashing.pkgproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <ItemGroup>
+    <ProjectReference Include="..\src\System.IO.Hashing.csproj">
+      <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
I think @ViktorHofer had some ideas for adding packages without also needing to add a .pkgproj. Viktor, what's the plan going forward for these?

Reply:
As System.IO.Hashing does neither use a reference assembly in the package nor a runtime specific asset you don't need the pkgproj anymore and can just set the two properties in the src project to make it packable:

```
<IsPackable>true</IsPackable>
<!-- TODO: Remove when the package ships with .NET 6. -->
<DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```